lib/autoload/send_file.rb

```diff
@@ -0,0 +1,19 @@
+module SendFile
+  # Makes the given file publicly accessible.
+  #
+  # @param [String] file The path to the local file.
+  # @param [String] public_name The file's public name, default set to file's base_name.
+  # @return [String] The url to the public visible file
+  def send_file(file, public_name = File.basename(file))
```

Comment:
and you can make this a module_function so you don't necessarily need to include the module